pkg/kubelet/kubelet_getters.go

```diff
@@ -274,6 +274,25 @@ func (kl *Kubelet) getPodVolumePathListFromDisk(podUID types.UID) ([]string, err
 	return volumes, nil
 }
 
+func (kl *Kubelet) getMountedVolumePathListFromDisk(podUID types.UID) ([]string, error) {
+	mountedVolumes := []string{}
+	volumePaths, err := kl.getPodVolumePathListFromDisk(podUID)
+	if err != nil {
+		return volumePaths, err
+	}
+	glog.V(4).Infof("Volume paths exist for pod %s: %v", podUID, volumePaths)
+	for _, volumePath := range volumePaths {
+		isNotMount, err := kl.mounter.IsLikelyNotMountPoint(volumePath)
+		if err != nil {
+			glog.Errorf("Cannot check whether volume path %s for pod %sis mounted or not: %v", volumePath, podUID, err)
```

Comment:
nit: missing a space `s/%sis/%s is`
Also change volume path printing to using `%q`

Reply:
Add `continue` to make sure you skip the next `if` check

Reply:
fixed